src/main/java/com/uid2/shared/attest/AttestationTokenRetriever.java

```diff
@@ -127,7 +131,7 @@ private void attestationExpirationCheck(long timerId) {
 
             attest();
         } catch (AttestationTokenRetrieverException | IOException e) {
```

Comment:
I know this was there before, but what would cause the IOException? Should we be returning a 401 for an IOException? Or is it a 500?

Reply:
httpClient.post() is what throws it. Changed this to return a 500

Reply:
Changed this to a 500 for an IOException, and fixed an issue in URLConnectionHttpClient.java that resulted in IOException being thrown for any non-200 status code